src/coreclr/vm/codeversion.h

```diff
@@ -287,28 +287,28 @@ class NativeCodeVersionNode
 #endif
 
 #ifdef FEATURE_ON_STACK_REPLACEMENT
-    PatchpointInfo * GetOSRInfo(unsigned * ilOffset);
+    PatchpointInfo * GetOSRInfo(unsigned * ilOffset) const;
 #endif
 
 private:
     //union - could save a little memory?
     //{
     PCODE m_pNativeCode;
-    PTR_MethodDesc m_pMethodDesc;
+    DAC_IGNORE(const) PTR_MethodDesc m_pMethodDesc;
     //};
 
-    ReJITID m_parentId;
+    DAC_IGNORE(const) ReJITID m_parentId;
     PTR_NativeCodeVersionNode m_pNextMethodDescSibling;
```

Comment:
There is a comment below saying m_pNextILVersionNode is never modified after being added to the list. I'm guessing the same thing holds here and we could add a similar comment?